src/NuGet.Clients/NuGet.PackageManagement.UI/PackageManagerToolWindowPane.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace NuGet.PackageManagement.UI
+{
+    [ProvideToolWindow(typeof(PackageManagerToolWindowPane), Style = VsDockStyle.Tabbed, DocumentLikeTool = true, Window = EnvDTE.Constants.vsCATIDDocument)]
+    public class PackageManagerToolWindowPane : ToolWindowPane, IVsWindowFrameNotify3
+    {
+        private PackageManagerControl _content;
+
+        /// <summary>
+        /// Initializes a new instance of the EditorPane class.
+        /// </summary>
+        public PackageManagerToolWindowPane(PackageManagerControl control)
+            : base(null)
+        {
+            _content = control;
+        }
+
+        public PackageManagerModel Model
+        {
+            get { return _content.Model; }
+        }
+
+        /// -----------------------------------------------------------------------------
+        /// <summary>
+        /// IVsWindowPane
+        /// </summary>
+        /// -----------------------------------------------------------------------------
+        public override object Content
+        {
+            get { return _content; }
+        }
+
+        private void CleanUp()
+        {
+            if (_content != null)
+            {
+                _content.CleanUp();
+                _content = null;
+            }
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            try
+            {
+                if (disposing)
+                {
+                    CleanUp();
+
+                    // Because Dispose() will do our cleanup, we can tell the GC not to call the finalizer.
+
+                    // TODO: Mirko - Not sure why this is necessary here but not in PpackageManagerWindowPane. Investigate.
```

Comment:
seems like the other one has an entry in globalsuppressions.cs:
[assembly: SuppressMessage("Build", "CA1816:PackageManagerWindowPane.Dispose(bool) calls GC.SuppressFinalize(object), a method that is typically only called within an implementation of 'IDisposable.Dispose'. Refer to the IDisposable pattern for more information.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.PackageManagement.UI.PackageManagerWindowPane.Dispose(System.Boolean)")]

(we recently turned on some analyzers in our build...and excluded old code for now).
Ideally, new code doesn't add a new suppression. Can you fix dispose pattern?

Reply:
Ah, that'd do it :) I checked the proj but didn't think to check there. I'll fix that up.